drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -217,7 +222,12 @@ end
 
 local function temp_attr_handler(driver, device, ib, response)
   if ib.data.value ~= nil then
-    local temp = utils.round(CONVERSION_CONSTANT/ib.data.value)
+    local temp_in_mired = ib.data.value
+    if (temp_in_mired < COLOR_TEMPERATURE_MIRED_MIN or temp_in_mired > COLOR_TEMPERATURE_MIRED_MAX) then
+      device.log.warn_with({hub_logs = true}, "Device reported color temperature outside of supported capability range")
+      temp_in_mired = utils.clamp_value(temp_in_mired, COLOR_TEMPERATURE_MIRED_MIN, COLOR_TEMPERATURE_MIRED_MAX)
```

Comment:
I think we probably just want to drop this on the floor, since it's probably not a _real_ value.

Reply:
i.e. just `return` after the log message